CMakeLists.txt

```diff
@@ -1,12 +1,13 @@
 #Copyright (c) 2022 Ultimaker B.V.
 #CuraEngine is released under the terms of the AGPLv3 or higher.
 
+cmake_policy(SET CMP0091 NEW)
 cmake_minimum_required(VERSION 3.20)
 project(CuraEngine)
 
 set(CURA_ENGINE_VERSION "master" CACHE STRING "Version name of Cura")
```

Comment:
Oop ... I know this isn't code you touched, but a great opportunity to scout master to main.